src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -1038,22 +1047,32 @@ get_user_reds(Reduction) ->
     element(2, Reduction).
 
 
+get_external_reds(Reduction) when tuple_size(Reduction) == 2 ->
```

Comment:
Minor nit, call this get_external_size_reds. external_reds could be a bit misleading here.